source/includes/_embed2.md.erb

```diff
@@ -442,6 +455,27 @@ Removes Embed2 from your page.
 
 Can be used to programatically open / close the Chat window. This method cannot be used with the `parentElement` option.
 
+#### triggerCampaign
+`triggerCampaign(campaignKey: string, options: object): Promise<void>;`
+
+Used to trigger the proactive Campaign with key `campaignKey`.
+
+An optional second argument `options` can be passed. These options may be helpful when testing your Campaign. The options are:
+`ignoreStatus?: boolean`: if `ignoreStatus` is `true`, the Campaign will be triggered even if it is inactive or in a draft state.
+`ignoreFrequency?: boolean`: if `ignoreFrequency` is `true`, the Campaign will be triggered even if it has already been triggered within the frequency configured on the Campaign.
```

Comment:
In the docs we also have `ignoreConditions`. Is that being used?

Reply:
I thought `triggerCampaign` never checks the conditions, and `evaluateCampaignConditions` always does

Reply:
Would you mind confirming in the code? I don't recall.

Reply:
I mean, that's how I implemented it :laughcry: I'm just saying I'm not sure if I did what was supposed to have been done, but that's what I did

Reply:
lol okay!
Then maybe just add a note that what's happening on page change is evaluateCampaignConditions + (maybe) triggerCampaign, and as such they can either trigger the evaluation whenever they need, or they can do their own evaluation in javascript and trigger directly?